src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -298,47 +314,51 @@ FROM <repository>/<image>:<tag>
+Docker privileged mode allows a container to have root-level access to the host from within the container. This enables containers to do more than they normally can but also presents security risks. You can enable your workspaces to have privileged mode, giving your users root-level access to the host where Che is running (in addition to root access of their workspaces). Privileged mode is necessary if you want to enable certain features such as Docker in Docker.
 
-By default, Che workspaces powered by a Docker container are not configured with Docker privileged mode. There are many security risks to activating this feature - please review the various issues with blogs posted online.
+By default, Che workspaces powered by a Docker container are not configured with Docker privileged mode. There are many security risks to activating this feature. Review the various issues with blogs posted online.
 
+To update the che.env variable:
 ----
-# Update your che.env:
 CHE_DOCKER_PRIVILEGED=true
 ----
 
 [id="mirroring-docker-hub"]
 == Mirroring Docker Hub
 
-If you are running a private registry internally to your company, you can https://docs.docker.com/registry/recipes/mirror/[optionally mirror Docker Hub]. Your private registry will download and cache any images that your users reference from the public Docker Hub. You need to https://docs.docker.com/registry/recipes/mirror[configure your Docker daemon to make use of mirroring].
+If you are running a private registry internal to your company, you can https://docs.docker.com/registry/recipes/mirror/[optionally mirror Docker Hub]. Your private registry will download and cache any images that your users reference from the public Docker Hub. You must https://docs.docker.com/registry/recipes/mirror[configure your Docker daemon to make use of mirroring].
 
 [id="using-docker-in-workspaces"]
-== Using Docker In Workspaces
+== Using Docker in workspaces
 
-If you’d like your users to work with projects which have their own Docker images and Docker build capabilities inside of their workspace, then you need to configure the workspace to work with Docker. You have three options:
+To allow your users to work with projects that have their own Docker images and Docker build capabilities inside of their workspaces, you must configure the workspaces to work with Docker. Following are the three options to configure the workspaces:
 
-1.  Activate Docker privileged mode, where your user workspaces have access to the host.
+* Activate the Docker privileged mode where your user workspaces have access to the host.
 
 ----
 # Update your codenvy.env to allow all Che workspaces machines/containers privileged rights:
 CHE_DOCKER_PRIVILEGED=true;
 ----
 
-1.  Configure Che workspaces to volume mount the host docker daemon socket file.
+* Configure Che workspaces to volume mount the host docker daemon socket file.
```

Comment:
Docker should be capitalized unless you're referring to the actual `docker` command.